content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -449,7 +450,11 @@ Each feature gate is designed for enabling/disabling a specific feature:
    for more details.
 - `CPUManager`: Enable container level CPU affinity support, see
   [CPU Management Policies](/docs/tasks/administer-cluster/cpu-management-policies/).
-- `CRIContainerLogRotation`: Enable container log rotation for cri container runtime.
+- `CRIContainerLogRotation`: Enable container log rotation for cri container runtime. Use kubelet flags `container-log-max-size` and `container-log-max-files`
+  to configure the maximum size for each log file and the maximum number of files allowed for each container respectively. The rotated log files can be accessed
```

Comment:
should talk about default/recommended values. Also behavior if nothing is set

Reply:
btw, pls spell cri as CRI.

Reply:
Fixed